cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -652,3 +656,54 @@ pluginConfig:
 		})
 	}
 }
+
+func TestPluginConfigV1Alpha1(t *testing.T) {
```

Comment:
This test should convert all the way from a file, in `TestSchedulerOptions`. See #87853

Otherwise we are just testing `Convert`, which should go in `conversion_test.go`. But you already have that test.

Reply:
Sure, I will move this logic to conversion_test.go.

Reply:
Maybe it's better to delete this test, and the test we need for this pr is already covered in conversion_test.go

Reply:
No, you need a test here (this file). It's just that it should simply be one new case of these https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/options/options_test.go#L248

Reply:
Sure, adding now.

Reply:
I have added some test logic here, reusing the existing plugin `TestSchedulerOptions/plugin_config` test.

Reply:
Never mind, I found I need to rebase my code. I will fix it,

Reply:
It's ok now, PTAL. 😄